src/Raven.Server/Commercial/SetupManager.cs

```diff
@@ -899,7 +899,14 @@ public static async Task AssertLocalNodeCanListenToEndpoints(SetupInfo setupInfo
             }
 
             var requestedEndpoints = localIps.ToArray();
-            var currentServerEndpoints = serverStore.Server.ListenEndpoints.Addresses.Select(ip => new IPEndPoint(ip, serverStore.Server.ListenEndpoints.Port)).ToArray();
+            var currentServerEndpoints = serverStore.Server.ListenEndpoints.Addresses.Select(ip => new IPEndPoint(ip, serverStore.Server.ListenEndpoints.Port)).ToList();
+
+            var tcpStatus = serverStore.Server.GetTcpServerStatus();
+            foreach (var listener in tcpStatus.Listeners)
+            {
+                if (IPAddress.TryParse(listener.LocalEndpoint.ToString(), out IPAddress ipAddress))
```

Comment:
Pretty sure that you can just cast this, no?

Reply:
no